src/udev/udev-builtin-net_id.c

```diff
@@ -326,14 +346,14 @@ static int dev_pci_slot(sd_device *dev, struct netnames *names) {
                         continue;
 
                 FOREACH_DIRENT_ALL(dent, dir, break) {
-                        unsigned i;
+                        int i;
                         char str[PATH_MAX];
                         _cleanup_free_ char *address = NULL;
 
                         if (dot_or_dot_dot(dent->d_name))
                                 continue;
 
-                        r = safe_atou_full(dent->d_name, 10, &i);
+                        r = safe_atoi(dent->d_name, &i);
```

Comment:
Hmm, would you mind splitting this out into a separate commit? It's changing the parsing a bit and I don't want to try to figure out a few years down the road if the change was intentional or accidental.